### PR TITLE
Fix npm vulnerabilities and update webpack everywhere except classic

### PR DESCRIPTION
**Pull request by NeOMakinG**
| Questions         | Answers
| ----------------- | -------------------------------------------------------
| Branch?           | 1.7.8.x
| Description?      | Our npm dependencies had a lot of vulnerabilities, it's mainly fixed by running `npm audit fix` and update webpack and all other dependencies (Notice that classic theme has not really been touched, another PR is opened for it)
| Type?             | bug fix
| Category?         | BO
| BC breaks?        | no
| Deprecations?     | no
| How to test?      | Not sure it has to be tested, but styles and JS in the BO should be as usually
| Possible impacts? | Assets build

---
This change is [Reviewable](https://reviewable.io/reviews/prestashop/prestashop/24572)

### CODE REVIEW

**Review by PierreRambaud — DISMISSED** [review]

**Review by Progi1984 — DISMISSED** [review]

**Comment by Progi1984** [comment]
@PrestaShop/qa-functional Do we send it to QA ? (Major changes versions for packages)

**Comment by NeOMakinG** [comment]
> @PrestaShop/qa-functional Do we send it to QA ? (Major changes versions for packages)

It's mainly dev dependencies, so the QA may test the asset build/watch and could do an otherall check on some pages, but no need to inspect everything I think

**Comment by Robin-Fischer-PS** [comment]
I think it would be best to have this one in QA indeed, we will test it before the Beta (as @NeOMakinG said, we will not test all the BO, just some pages.

File: admin-dev/themes/default/webpack.config.js [hunk 1]
```diff
@@ -95,7 +96,7 @@ module.exports = (env, argv) => {
   if (!devMode) {
     config.optimization.minimizer = [
       new UglifyJsPlugin({
-        sourceMap: false,
+        sourceMap: true,
```
Comment (Progi1984) [inline admin-dev/themes/default/webpack.config.js]: sourcemap in production mode ?

**Review by Progi1984 — COMMENTED** [review]
A small comment

**Review by Progi1984 — DISMISSED** [review]

**Review by Progi1984 — APPROVED** [review]

**Review by PierreRambaud — APPROVED** [review]

**Comment by Robin-Fischer-PS** [comment]
Hi @NeOMakinG !

@SD1982 did a BO full automated test campaign, and I didManual navigation on some pages on BO, both have shown no defect in behavior or display.

So it's QA :heavy_check_mark: , thanks !

**Comment by matks** [comment]
Thank you @NeOMakinG

**Pull request merged**